platform-python.spec

```diff
@@ -472,131 +388,83 @@ BuildRoot: %{_tmppath}/%{name}-%{version}-root
 
 # See notes in bug 532118:
-Provides: python(abi) = %{pybasever}
+Provides: platform-python(abi) = %{pybasever}
 
 Requires: %{name}-libs%{?_isa} = %{version}-%{release}
 
-# In order to support multiple python interpreters, apart from the system python3,
-# for development purposes, new packages were introduced which can be installed in parallel
-# with the main python3 package (e.g. 1369688), with the naming scheme 'python<version>',
-# however in order to keep the upgrade path clean we need to Obsolete and Provide
-# these packages at the main python3 package.
-Obsoletes: python%{pyshortver}
-Provides: python%{pyshortver} = %{version}-%{release}
-
-%if 0%{with_rewheel}
-Requires: python3-setuptools
-Requires: python3-pip
-%endif
-
 %description
-Python is an interpreted, interactive, object-oriented programming
-language often compared to Tcl, Perl, Scheme or Java. Python includes
-modules, classes, exceptions, very high level dynamic data types
-and dynamic typing. Python supports interfaces to many system calls and
-libraries, as well as to various windowing systems (X11, Motif, Tk,
-Mac and MFC).
-
-Programmers can write new built-in modules for Python in C or C++.
-Python can be used as an extension language for applications that
-need a programmable interface.
-
-Note that documentation for Python is provided in the python3-docs package.
-
-This package provides the "python3" executable; most of the actual
-implementation is within the "python3-libs" and "system-python-libs" packages.
+Python interpreter for low-level system tools, designed to work with
+the Platform module.
 
 %package libs
-Summary:        Python runtime libraries
-Group:          Development/Libraries
-# For Modularity purpose we need not to include the dist-tag int he dependency
-%if %(d="%{?dist}"; [ "${d#module-base-runtime-}x" != "${d}x" ] && echo 1 || echo 0)
-Requires:       system-python-libs%{?_isa} = %{version}
-%else
-Requires:       system-python-libs%{?_isa} = %{version}-%{release}
-%endif
+Summary:        Platform Python runtime libraries
 
 # expat 2.1.0 added the symbol XML_SetHashSalt without bumping SONAME.  We use
 # this symbol (in pyexpat), so we must explicitly state this dependency to
 # prevent "import pyexpat" from failing with a linker error if someone hasn't
 # yet upgraded expat:
 Requires: expat >= 2.1.0
-Provides: python3-enum34 = 1.0.4-5%{?dist}
-Obsoletes: python3-enum34 < 1.0.4-5%{?dist}
-
-%description libs
-This package contains runtime libraries for use by Python:
-- the libpython dynamic library, for use by applications that embed Python as
-a scripting language, and by the main "python3" executable
-- the Python standard library
 
-%package -n system-python
-Summary:        System Python executable
-Group:          Development/Libraries
-Requires:       system-python-libs%{?_isa} = %{version}-%{release}
-Provides:       system-python(abi) = %{pybasever}
+%define __requires_exclude ^(/usr/bin/python3.*|python\\(abi\\) = 3\\..*)$
```

Comment:
While I agree about both points, I think it's safer to just leave it as is as a safety measure.